app/views/admin/edition_images/_image_upload.html.erb

```diff
@@ -9,9 +9,9 @@
 ) do %>
   <%= render "govuk_publishing_components/components/file_upload", {
     name: "image[image_data][file]",
-    id: "edition_images_image_data_file",
+    id: "edition_images_file",
```

Comment:
It's unclear to me why this has changed

Reply:
Ah yea this is a hang-up from the previous solution, I forgot to change this back after Ollie's review. Apologies!